web_monitoring/diffing_server.py

```diff
@@ -89,10 +92,22 @@ def get(self, differ):
 
     def write_error(self, status_code, **kwargs):
         response = {'code': status_code, 'error': self._reason}
+
+        # Handle errors that are allowed to be public
+        actual_error = 'exc_info' in kwargs and kwargs['exc_info'][1] or None
+        if (isinstance(actual_error, UndiffableContentError) or
```

Comment:
`isinstance(actual_error, (UndiffableContentError, UndecodableContentError))` would also work here

Reply:
OH! I didn’t even realize that could take a sequence as the second argument. 👍